docker/girder.local.conf

```diff
@@ -1,3 +1,6 @@
 [global]
 tools.proxy.on: True
 server.socket_host = "0.0.0.0"
+
+[server]
+mode = "development"
```

Comment:
Can this be specified as a command link option? It would be nice to be able to switch between dev and production.

Reply:
Unfortunately, it doesn't look like this is a command line option to me as far as I can tell.